src/ado2gh/Program.cs

```diff
@@ -47,19 +47,29 @@ public static async Task Main(string[] args)
 
             SetContext(parser.Parse(args));
 
+            WarnIfNotUsingExtension();
+
             try
             {
                 await LatestVersionCheck(serviceProvider);
             }
             catch (Exception ex)
             {
-                Logger.LogWarning("Could not retrieve latest ado2gh version from github.com, please ensure you are using the latest version by downloading it from https://github.com/github/gh-gei/releases/latest");
+                Logger.LogWarning("Could not retrieve latest ado2gh CLI version from github.com, please ensure you are using the latest version by running: gh extension upgrade ado2gh");
                 Logger.LogVerbose(ex.ToString());
             }
 
             await parser.InvokeAsync(args);
         }
 
+        private static void WarnIfNotUsingExtension()
+        {
+            if (!AppContext.BaseDirectory.EndsWith("extensions\\gh-ado2gh\\") && !AppContext.BaseDirectory.EndsWith("extensions/gh-ado2gh/"))
```

Comment:
I'd use `Path.Combine`:
```suggestion
            if (!AppContext.BaseDirectory.EndsWith(Path.Combine("extensions", "gh-ado2gh").ToString())
```

Reply:
I'm not sure that will add the trailing slash properly, I'll try it locally in a min